features/fixtures/sampler/android/app/src/main/java/com/sampler/MainApplication.java

```diff
@@ -43,6 +49,21 @@ public ReactNativeHost getReactNativeHost() {
   @Override
   public void onCreate() {
     super.onCreate();
+    boolean enableANR = false;
+    int anrTimeout = 0;
+    try {
+      ApplicationInfo ai = getPackageManager().getApplicationInfo(getPackageName(), PackageManager.GET_META_DATA);
+      String rawEnableANR = (String) ai.metaData.get("enable-anr");
+      enableANR = Boolean.parseBoolean(rawEnableANR);
+      anrTimeout = Integer.parseInt((String) ai.metaData.get("anr-timeout"));
+    } catch (NameNotFoundException e) {
```

Comment:
We should rethrow this so that the error will be recorded in the maze_output directory, as this will make debugging scenarios much easier.